azure/eventhub_log_forwarder/event_hub.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "namespaceName": {
```

Comment:
I approved this but one more comment - can you make this `eventhubNamespace` to match the above? I also have it called this in my template, and it'll need to be the same as all of the templates being used by the parent template

Reply:
good catch! I've changed that now